### DISCUSSION TITLE
Detaching a fork into a private repo - est time to service ticket

### DISCUSSION THREAD

**Discussion by yoni-lavi** [community/community#59046] [Repositories]
### Select Topic Area

Question

### Body

- As in the title, how long should I expect to wait before such ticket be serviced?
- Why does taking a fork of a public repo and converting it to a standalone repo require GitHub staff approval in the first place?
- Separately, why can't a fork of a public repo be set to private visibility?

**Comment by SamSunshine** [comment]
_The following answer was generated with AI, hope it helps:_ 

The time it takes for a support ticket to be serviced by GitHub can vary depending on several factors, including the volume of support requests they receive and the complexity of the issue. GitHub aims to provide timely assistance to their users, but the specific response time can't be guaranteed. For urgent or critical issues, it's recommended to reach out to GitHub support directly to inquire about the estimated response time.

Regarding the requirement for GitHub staff approval when converting a fork of a public repository into a standalone repository, this is a security measure to prevent abuse or unauthorized actions. Converting a fork into a standalone repository detaches it from the original repository's network, potentially removing connections, dependencies, and links to the original repository. GitHub wants to ensure that such actions are deliberate and authorized by the repository owner.

As for setting the visibility of a forked repository to private, it's important to note that forks inherit the visibility of the original repository. This ensures that any contributions made to the original repository are also shared with the community. By default, public repositories can only be forked as public repositories. If you need to have a private version of a forked repository, you can manually create a new private repository and then clone the forked repository to your local machine, making it a separate private repository.
